JPS_Ontology/ontology/ontoassetmanagement/README.MD

```diff
@@ -186,7 +189,40 @@ The ontology can be divided into these respective domains:
         hasCurrentLocation string
     }
 ```
-### 3.8. Furniture and Workspace
+### 3.8. Technical System
+```mermaid
+    erDiagram
+    "ontocape_technical_system:TechnicalSystem" ||--o{ "bot:Element" : isStoredAt
```

Comment:
The properties are in ontoassetmanagement. https://github.com/cambridge-cares/TheWorldAvatar/blob/674e7d02cde79438086f0f1c4baccef935b092f2/JPS_Ontology/ontology/ontoassetmanagement/OntoAssetManagement.owl#L741 

The reason is because we have assets being stored at other assets. For example, based on the asset data that we have recorded, we have some smaller items that are being stored at the larger items thus we require this property.

Reply:
So the purpose of including ontocape_technical_system:TechnicalSystem is because it is possible that we might have some element that is being stored at some TechnicalSystem such as a LabEquipmentSystem or vice versa.